.github/workflows/helm-docs-validation.yaml

```diff
@@ -0,0 +1,28 @@
+name: Helm Docs Validation
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  helm_docs_validation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      
+      - name: Install helm-docs
+        run: |
+          curl -LO https://github.com/norwoodj/helm-docs/releases/download/v1.11.0/helm-docs_1.11.0_Linux_x86_64.tar.gz
+          tar -xvf helm-docs_1.11.0_Linux_x86_64.tar.gz
+          sudo mv helm-docs /usr/local/bin/
+
+      - name: Generate Helm Docs
+        run: make generate-helm-docs
+
+      - name: Check for file changes
+        run: |
```

Comment:
maybe it's good to add `shell: bash`

Reply:
It's not critical production code, and it gets the job done, so I think it's fine.
what does `shell: bash` do?